sycl/source/detail/device_image_impl.hpp

```diff
@@ -256,7 +256,7 @@ class device_image_impl {
           // supposed to be called from c'tor.
           MSpecConstSymMap[std::string{SCName}].push_back(
               SpecConstDescT{/*ID*/ It[0], /*CompositeOffset*/ It[1],
-                             /*Size*/ It[2], BlobOffset, HasDefaultValues});
+                             /*Size*/ It[2], BlobOffset});
```

Comment:
I'm a bit concerned that this change is being committed in scope of this PR. The PR title says that this is a modification of `sycl-post-link` tool and all 13 commits are very likely to be squashed into a single one during the merge - we will end up with a commit to `sycl-post-link`, which modifies DPC++ RT.

Moreover, it is not clear to me why this change is needed at all. From my point of view it is better if we submit it in a separate PR

Reply:
Tagging @dm-vodopyanov here

Reply:
This change is related to https://github.com/intel/llvm/pull/3667/files#r631154823. I'll create a separate PR to make it clear.